lib/puppet_litmus/serverspec.rb

```diff
@@ -138,14 +138,14 @@ def bolt_upload_file(source, destination, options = {})
   # @param task_name [String] The name of the task to run.
   # @param params [Hash] key : value pairs to be passed to the task.
   # @return [Object] A result object from the task.
```

Comment:
and add in what new values are available in the result object.

Reply:
@tphoney updated the result section. The values available are stdout, stderr and result